app/coffee/DocManager.coffee

```diff
@@ -30,9 +30,9 @@ module.exports = DocManager =
 				else
 					callback err, doc
 
-	getAllDocs: (project_id, callback = (error, docs) ->) ->
+	getAllNonDeletedDocs: (project_id, callback = (error, docs) ->) ->
 		DocArchive.unArchiveAllDocs project_id, (error) ->
```

Comment:
Only because we generally have a project all archived, or all unarchived. Splitting this would be more complex.